osgar/lib/config.py

```diff
@@ -14,7 +18,20 @@ class Config(object):
 
     @classmethod
     def load(cls, filename):
-        return cls.loads(open(filename).read())
+        if isinstance(filename, list):
+            filenames = filename
+        else:
+            filenames = [filename]
+        
+        ret = Config()
+        for filename in filenames:
+            with open(filename) as f:
+                c = cls.loads(f.read())
```

Comment:
I forgot how much I don't like this whole class. The class method `loads` stores loaded data in a class attribute. Now you call it multiple times so only data from the last call are stored in `cls.data`. I don't really understand why.